RecoTracker/DeDx/plugins/SiPixelTrackProbQXYProducer.cc

```diff
@@ -161,28 +163,34 @@ void SiPixelTrackProbQXYProducer::produce(edm::StreamID id, edm::Event& iEvent,
     probQonTrackNoLayer1 = probQonTrackWMultiNoLayer1 * probQonTrackTermNoLayer1;
     probXYonTrackNoLayer1 = probXYonTrackWMultiNoLayer1 * probXYonTrackTermNoLayer1;
 
-    indices.push_back(resultSiPixelTrackProbQXYColl->size());
-    resultSiPixelTrackProbQXYColl->emplace_back(
-        probQonTrack, probXYonTrack, probQonTrackNoLayer1, probXYonTrackNoLayer1);
+    // Store the values in the collection
+    resultSiPixelTrackProbQXYColl->emplace_back(SiPixelTrackProbQXY(probQonTrack, probXYonTrack));
```

Comment:
Ah maybe this is wrong? But resultSiPixelTrackProbQXYColl is a collection of SiPixelTrackProbQXY objects, so I thought this should work